AzureMS/scripts/npc/notcoded.js

```diff
@@ -2,7 +2,7 @@ function action(mode, type, selection) {
 	if (cm.getNpc() >= 9901000) {
 		cm.sendNext("Hello #h0#, I am in the Hall of Fame for reaching LEVEL 200.");
 	} else {
-		cm.sendNext("This Npc ("+cm.getNpc()+") is shutdown!\r\nSouls needs to code faster sometimes :^)\r\n\r\n- SoulGirlJP.");
+		cm.sendNext("This Npc ("+cm.getNpc()+") is shutdown!\r\n Note to Future self: Duck needs to code this aaaaa");
```

Comment:
Consider changing the message to say that the NPC is not coded yet - we've had people asking how to "turn NPCs in Azure on", since the word "shutdown" was used.